apps/src/dom.js

```diff
@@ -30,7 +30,10 @@ var addEvent = function(element, eventName, handler) {
   var touchEvent = exports.getTouchEventName(eventName);
   if (touchEvent) {
     element.addEventListener(touchEvent, function(e) {
-      e.preventDefault();  // Stop mouse events.
+      // Stop mouse events and suppress default event handler to prevent
+      // unintentional double-clicking
+      e.preventDefault();
+      element.removeEventListener(eventName, handler);
```

Comment:
By removing the event listener, dont we make it so that if you were to want to touch this element a second time, it wouldn't work the second time? I believe there are some other places where we prevent unintentional double clicks by using "debounce" (which I think does something along the lines of ignore this event if it just happened in the last n ms)